CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+:warning: :warning: :warning:
+
+**IMPORTANTE:** podés cargar tus sugerencias en español o en inglés, ¡usá el idioma en el que te puedas expresar mejor!
+
+**IMPORTANT:** you can upload issues both in english and spanish, choose the language that better helps you to express yourself!
+
+:warning: :warning: :warning:
+
+# ![Argentina](http://mumuki.io/static/argentina.png) Español
+
+¡Hola! Antes que nada, queremos agradecerte por la contribución que estás por hace; tener una comunidad proponiendo cambios y mejoras es la única forma de que la plataforma mejore. Para entendernos mejor, te pedimos por favor que prestes atención a los siguientes puntos:
```

Comment:
I'd add some emojis to keep this contribution guidelines informal and following mumuki wording style